src/devtools/client/debugger/dist/vendors.js

```diff
@@ -4722,7 +4722,7 @@ var _react = __webpack_require__(6);
 
 var _react2 = _interopRequireDefault(_react);
 
-var _reactDom = __webpack_require__(112);
```

Comment:
this is no longer needed because, we're transitioning away from vendors.js, but it is nice to have incase we want to revert back to this artifact later.